paymentsheet/src/main/java/com/stripe/android/paymentsheet/LinkHandler.kt

```diff
@@ -110,11 +109,6 @@ internal class LinkHandler @Inject constructor(
 
     fun prepareLink(state: LinkState?) {
         setupLink(state)
-
-        if (state?.isReadyForUse == true) {
-            // If account exists, select Link by default
-            savedStateHandle[SAVE_SELECTION] = PaymentSelection.Link
-        }
```

Comment:
Are we certain this is the UX we desire? We don't want to show link as the selected payment method if an account exists?

Reply:
Yes. We should move all automated selection out of the view models and into the `PaymentSheetLoader`. Otherwise, we end up with discrepancies between what `flowController.getPaymentOption()` returns and what the payment sheet shows.

Reply:
This is the current behavior. I just want to be sure that this is the strategy we want. I understand that it might not make sense, but we should be careful in terms of changing the UX here.